### PR TITLE
Aric improve email

### PR DESCRIPTION
**Pull request by AricStewart**

### CODE REVIEW

File: functions/functions.inc [hunk 1]
```diff
@@ -262,3 +259,19 @@ function output_csv($report, $filename)
     }
 
 }
+
+function urlsafe_b64decode($string) {
```
Comment (mshappe) [inline functions/functions.inc]: You define these here, and then again, very differently, in common...

Comment (AricStewart) [reply functions/functions.inc]: One is PHP and one is Javascript.

Comment (AricStewart) [reply functions/functions.inc]: One is PHP and one is Javascript.

Comment (mshappe) [reply functions/functions.inc]: ...derp *facepalm*

**Pull request merged**